server.py

```diff
@@ -1,10 +1,11 @@
-from fastapi import FastAPI, Request
-from fastapi.responses import JSONResponse, RedirectResponse
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
 import uvicorn
 import sqlite_helpers as helpers
 from fastapi.testclient import TestClient
+from hash import create_alias
 from args import get_args
-import hash
+from enums import HttpStatus
```

Comment:
based on the enums change i mentioned but it is up to you if you want to leave it inside the class
```suggestion
from enums import HttpStatus, code_to_enum
```